packages/debugger/src/service.ts

```diff
@@ -216,6 +216,29 @@ export class DebuggerService implements IDebugger, IDisposable {
     return { ...reply.body, path: source.path ?? '' };
   }
 
+  /**
+   * Evaluate an expression.
+   *
+   * @param expression The expression to evaluate as a string.
+   */
+  async evaluate(expression: string): Promise<void> {
+    if (!this.session) {
+      throw new Error('No active debugger session');
+    }
+    const frameId = this.model.callstack.frame?.id;
+    const reply = await this.session.sendRequest('evaluate', {
+      context: 'repl',
+      expression,
+      frameId
+    });
+    if (!reply.success) {
+      return;
+    }
+    // get the frames to retrieve the latest state of the variables
+    this._clearModel();
+    await this._getAllFrames();
+  }
```

Comment:
```suggestion
  
      return reply;
  }
```

So the first step towards implementing printing of the eval output will be to add a return value this `evaluate` method. According to the [DAP spec](https://microsoft.github.io/debug-adapter-protocol/specification#Requests_Evaluate), `reply.result` should hold the result of the eval. Since there are also other printing-related params in `reply` (such as `.presentationHint`), I think it makes sense to just return the whole response if the request suceeds